quickstart/scripts/start-docker-desktop.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+# Function to check if Docker is running
+is_docker_running() {
+  if ! timeout 5 docker info > /dev/null 2>&1; then
+    echo "Docker is not responding within the timeout period."
+    return 1
+  fi
+  return 0
+}
+
+echo "Checking if Docker is running..."
+
+# Check if Docker is running
+if ! is_docker_running; then
+  echo "Docker is not running. Starting Docker Desktop..."
+  open -a Docker
```

Comment:
This looks specific to desktop app deployments (likely, the Mac users), where the docker daemon is implicitly launched via the desktop app?

What if docker is installed with a different setup (for example, for a Linux user)?

Rather than taking ownership of the "how to run docker" problem, would it be simpler to de-scope this script to just tell the user that docker doesn't seem to be running and error-out with a hint?

Reply:
Yeah, that's a good point, the script might not work on some Linux OS.


Note that this script is from [canton-network-utility](https://github.com/DACH-NY/canton-network-utilities/blob/main/scripts/start-docker-desktop.sh)
I quite like it when running the utility app, thus propose to do it here. 
Maybe we can try and add:
1. command to start it in Linux
2. some more protection in case the "start does not work on specific OS".
"Try starting by running <command> failed, please start Docker service manually."

Reply:
I think this is covered here: https://docs.docker.com/engine/daemon/start/

```
$ sudo systemctl start docker
```

Looks like there is no unified way to start it - needs an OS check :(

Reply:
updated the script to support Linux and Windows to start docker service.
I have only tested it on my MacOS, wondering if people can help testing on other OS?
Note that for Windows, it won't work if it's not ran in a Unix-like subsystem like Git-bash, WSL.

Reply:
I believe @iggy-da uses Ubuntu.